src/TTimer.cpp

```diff
@@ -91,7 +91,6 @@ void TTimer::setName(const QString& name)
 
 void TTimer::setTime(QTime time)
 {
-    QMutexLocker locker(&mLock);
```

Comment:
I don't think that is the case.

Reply:
Maybe Heiko imagined that eventually it would be possible to run the behind the GUI stuff in separate threads (for each profile) - which would have been nice if it was doable but a *whole* new (and very thick!) level of complexity I guess... 🤷‍♂️

Reply:
Yeah... not one that turns out we needed!